erpnext/stock/doctype/purchase_receipt/purchase_receipt.py

```diff
@@ -467,25 +465,11 @@ def update_billing_status(self, update_modified=True):
 
 		self.load_from_db()
 
-	def create_package_tag(self):
+	def validate_duplicate_package_tags(self):
 		package_tags = [item.package_tag for item in self.items if item.package_tag]
```

Comment:
You should also check if the package tag is already linked with an existing item or batch